src/arrayMethodJoin.js

```diff
@@ -4,8 +4,21 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let string = '';
+
+    for (let i = 0; i < this.length; i++) {
+      if (this[i] === null || this[i] === undefined) {
+        this[i] = '';
+      }
```

Comment:
Let do action if all good
```suggestion
      if (this[i] !== null && this[i] !== undefined) {
        string += this[i];
      }
```